README.md

```diff
@@ -422,6 +422,30 @@ done manually by someone on the Azul team. Ask on Slack to find out more.
 
 [Terra Data Repository]: https://jade.datarepo-dev.broadinstitute.org/
 
+### 3.2.2 Personal account registration
+Peronsal accounts may want to be registered with SAM to allow access to TDR,
+or for registration as an administrator for TDR. In order to register your 
+personal email with SAM the following actions need to be performed.
```

Comment:
This is where I need to be a grammar nerd. An account can't "want" anything; only people can have wants or desires. I would reword this to say "Personal accounts may need to be registered"